src/test/suite/commands/runDraftTool/helper/draftCommandBuilder.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as assert from 'assert';
+import {
+   buildCreateCommand,
+   buildCreateConfig,
+   buildSetupGHCommand,
+   buildGenerateWorkflowCommand,
+   buildUpdateIngressCommand
+} from '../../../../../commands/runDraftTool/helper/draftCommandBuilder';
+import * as path from 'path';
+
+suite('Draft Command Builder Test Suite', () => {
+   test('it builds create command for dockerfile', () => {
+      const testDestination = './';
+      const fileType = 'dockerfile';
+      const configPath = './testConfig';
+
+      assert.strictEqual(
+         buildCreateCommand(testDestination, fileType, configPath),
+         `create --skip-file-detection --destination=${testDestination} --dockerfile-only --create-config=${configPath}`
+      );
+   });
+   test('it builds create command for deployment file', () => {
+      const testDestination = './';
+      const fileType = 'deployment';
+      const configPath = './testConfig';
+
+      assert.strictEqual(
+         buildCreateCommand(testDestination, fileType, configPath),
+         `create --skip-file-detection --destination=${testDestination} --deployment-only --create-config=${configPath}`
+      );
+   });
+   test('it returns the create config file name', () => {
+      const lang = 'go';
+      const port = '8080';
+      const appName = 'testApp';
+      const workflow = 'testWorkflow';
+      const langVersion = '1.18';
+      const namespace = 'testNamespace';
+      const imageName = 'testImage';
+      const imageTag = '0.0.1';
+      var result = buildCreateConfig(
+         lang,
+         port,
+         appName,
+         workflow,
+         langVersion,
+         namespace,
+         imageName,
+         imageTag
+      );
+      assert.match(result, /^.*\.(yaml)$/);
```

Comment:
we should check the actual contents of the create config yaml file here as well

Reply:
added a validation for the yaml contents in the file.